package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "author": "vyzo",
+  "bugs": {},
+  "gx": {
+    "dvcsimport": "github.com/libp2p/go-libp2p-autonat"
+  },
+  "gxDependencies": [
+    {
+      "author": "whyrusleeping",
+      "hash": "QmPL3AKtiaQyYpchZceXBZhZ3MSnoGqJvLZrc7fzDTTQdJ",
+      "name": "go-libp2p",
+      "version": "6.0.19"
+    }
```

Comment:
Seems to be missing a few deps, but since this needs to be split anyways I guess it's fine

Reply:
yeah, the rationale was that all the deps get pulled by go-libp2p, but it will be split and not depend on that at all.